src/transport_ofi.c

```diff
@@ -1214,7 +1214,7 @@ int shmem_transport_startup(void)
 int shmem_transport_fini(void)
 {
     /* Wait for acks before shutdown */
-    shmem_transport_quiet();
+    shmem_transport_quiet(SHMEM_CTX_DEFAULT);
```

Comment:
Once you introduce ```shmem_transport_ctx_default```, you'll want to use that here in place of ```SHMEM_CTX_DEFAULT```.  Otherwise, the types won't match (```SHMEM_CTX_DEFAULT``` is ```void *``` and ```shmem_transport_ctx_default``` should be of type ```shmem_transport_ctx_t *``` ).

If you make the type change I suggested, you'll want to also introduce the new variable in this PR to resolve the type issue.